solutions/USACO jan18-billboard.mdx

```diff
@@ -61,3 +122,85 @@ elif brCorner and blCorner:
 
 </PySection>
 </LanguageSection>
+
+## Solution 2
+
+As the problem statement says, the remaining cow feed billboard is situated in front of the lawnmower billboard, potentially obscuring it; therefore, we can split into six cases to consider. 
+
+I will use the images to visualize the cases, so note these things: The red color shows the area covered by the second recangle (whose borders are black), while the green one represents the area of the first rectangle (their borders are blue). If two rectangles intersect, the red area will appear as the cow feed billboard block the lawnmover billboard. 
+
+Our task is to calculate the area of the first rectangle not obscured by the second one (which is the green area as I described). There are 6 cases to consider, which are illustrated below:
+
+### Case 1
+
+![](<USACO-jan18-billboard-bronze/Case_1.1.png>)
+
+![](<USACO-jan18-billboard-bronze/Case_1.2.png>)
+
+### Case 2
+
+![](<USACO-jan18-billboard-bronze/Case_2.png>)
+
+### Case 3
+
+![](<USACO-jan18-billboard-bronze/Case_3.png>)
+
+### Case 4
+
+![](<USACO-jan18-billboard-bronze/Case_4.png>)
+
+### Case 5
+
+![](<USACO-jan18-billboard-bronze/Case_5.png>)
+
+### Case 6
+
+![](<USACO-jan18-billboard-bronze/Case_6.png>)
+
+The 6-th case also includes a corner case where two rectangles intersect with their corners, image below:
+
+![](<USACO-jan18-billboard-bronze/Corner_Case.png>)
+
+<LanguageSection>
+
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+#include <iostream>
+#include <cstdio>
+using namespace std;
+
+int main(){
+	ios_base::sync_with_stdio(0); cin.tie(0);
+	freopen("billboard.in","r",stdin);
+	freopen("billboard.out","w",stdout);
+	int x[5],y[5];
+	for (int i=1;i<=4;++i) {
+		cin >> x[i] >> y[i];
+	} 
+	if (x[4]>=x[2] && x[3]<=x[1] && y[4]>=y[2] && y[3]<=y[1]) {
```

Comment:
```suggestion
	if (x[4]>=x[2] && x[3]<=x[1] && y[4]>=y[2] && y[3]<=y[1]) { // Case 1
```

Do you think it would help to enumerate the cases? (like the way shown in suggestion)